ARCHITECTURE.md

```diff
@@ -24,16 +24,18 @@ See [Prisma Generators](https://prismaio.notion.site/Prisma-Generators-a2cdf2622
 
 ## The `DMMF`, or Data Model Meta Format
 
-What the ... is DMMF? It's the Datamodel Meta Format. It is an AST (abstract syntax tree) of the datamodel in the form of JSON.
+What the ... is DMMF? It's the Datamodel Meta Format. It is an AST (abstract syntax tree) of the datamodel in the form of JSON.  
```

Comment:
Is the double space intended?

Reply:
Probably yes to have a line return

Reply:
Yep, the document was written as if someone originally intended there to be line breaks, but GitHub Markdown does only render them with double spaces - so I added them. Alternative would have been to reformat the file itself to not have line breaks that do not render.